src/app/[locale]/(components)/cartIconWithCount.tsx

```diff
@@ -2,14 +2,17 @@ import { getCheckoutID } from "@/core/server/checkout";
 import { gqlClient } from "@/gql";
 import { Suspense } from "react";
 import { CartIcon } from "./cartIcon";
+import { getTranslations } from "@/core/server/getTranslations";
 
 async function AsyncCartIcon({
   promise,
 }: {
   promise: Promise<number | undefined>;
 }) {
+  const t = await getTranslations();
+  const a11y = {cartIcon: t("header.a11y.cartIcon")}
   const quantity = await promise;
-  return <CartIcon quantity={quantity} />;
+  return <CartIcon quantity={quantity} a11y={a11y}/>;
```

Comment:
I think we should avoid creating a new pattern, I don't see this anywhere else in the codebase. I would rather we pass `aria-label` if possible, or `ariaLabel` if for some reason the former isn't possible.

Reply:
Its going to be a patter, that if we put it on the a11y dictonary its for all a11y stuff